modules/ROOT/pages/logging/security-log-analyzer.adoc

```diff
@@ -0,0 +1,170 @@
+:log-retention-days: 30
+:max-request-hours: 24
+
+[[aura-monitoring]]
+= Security log analyzer
+
+label:AuraDB-Professional[]
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Security log analyzer is a feature that provides a UI to review the security events on an Aura instance.
+
+You can access security logs from *Logs* in the left-hand navigation and selecting *Security* in the tabs at the top.
+
+To toggle between instances, use the dropdown menu on the top left.
+
+Security log analyzer is split up in two parts:
+
+* *Summary table* - An aggregated view of security logs, giving a high level overview over the selected time period.
+* *Details table* - A detailed view showing individual security events in the selected time period.
+
+To fetch logs, press the *Fetch logs* button. 
+You may optionally select any filters or search text if required, then press *Fetch*.
```

Comment:
Maybe also mention selecting time window here. Either that, or leave both filters/time window to the section below.